_docs/Architectural-Overview.md

```diff
@@ -1,34 +1,29 @@
 ---
 title: Architectural Overview
 ---
-![architectural_overview.png](/images/architectural_overview.png)
+# **[To be replaced by stack description]**
 
 # Client
 
 ## Core
-* [Bridgehead](../Components-Bridgehead/)
-  * Interface between modules and Qabel servers. This is the only component implementing Qabel protocols.
+* [Platform Interface](../Components-Platform-Interface/)
+  * Interface between modules and Qabel servers.
 * Util
-  * Utility functions (which don't need network), e.g. encoding / decoding messages.
+  * Utility functions (which do not need network), e.g. encoding / decoding messages.
 * Config
   * Access to the Qabel client configuration, e.g. various user settings.
 * [Contacts](../Components-Contacts/)
   * Access to the Qabel client users address book.
 
 ## Modules
-Provide the principal functionality end-users expect of the Qabel client. E.g. Mail, XMPP chat, file sync.
-
-* File sync
-  * Allows end-users to define folders which will be synced across their devices.
-* Mail
-  * Local SMTP and IMAP server enabling the end-user to send and receive mails using a traditional mail application. The application data is transmitted using the Qabel protocols.
+Will provide additionally functionality after BETA. E.g. Mail, XMPP chat, CalDAV.
```

Comment:
We should delete this whole section until we actually work on providing functionality for things which one might call a `module`.